File: src/Neuralm.Presentation.Web/src/messages/responses/RegisterResponse.ts
import Response from './Response';

/**
 * Represents the RegisterResponse class.
 */
export default class RegisterResponse extends Response {
  /**
   * Initializes a new instance of the RegisterResponse class.
   */
  constructor(id: string, requestId: string, dateTime: Date, message: string, success: boolean) {
    super(id, requestId, dateTime, message, success);
  }
}

File: src/Neuralm.Services/Neuralm.Services.UserService/Neuralm.Services.UserService.Messages/RegisterResponse.cs
using Neuralm.Services.Common.Messages.Abstractions;
using System;

namespace Neuralm.Services.UserService.Messages
{
    /// <summary>
    /// Represents the <see cref="RegisterResponse"/> class.
    /// </summary>
    public class RegisterResponse : Response
    {
        /// <summary>
        /// Initializes an instance of the <see cref="RegisterResponse"/> class.
        /// </summary>
        /// <param name="requestId">The request id.</param>
        /// <param name="message">The message.</param>
        /// <param name="success">The success flag.</param>
        public RegisterResponse(Guid requestId, string message = "", bool success = false) : base(requestId, message, success)
        {

        }

        /// <summary>
        /// Initializes a new instance of the <see cref="RegisterResponse"/> class.
        /// SERIALIZATION CONSTRUCTOR!
        /// </summary>
        public RegisterResponse()
        {
            
        }
    }
}
